npm-deps.txt

```diff
@@ -1 +1,3 @@
 dockerfilelint
+dockerfile_lint
+
```

Comment:
I don't think we need this blank line

Reply:
6e4387c664528a1ec1ce42c8745ad30feee955e9